src/data/klage-eller-anke-temaer.ts

```diff
@@ -0,0 +1,59 @@
+export interface KategoriTema {
+    tema: string;
+    tittel: string;
+    skjemaveileder: boolean;
+    digital: boolean;
+    bareAnke: boolean;
+}
+
+export const KLAGE_ELLER_ANKE_TEMAER = [
+    {
+        tittel: 'Arbeid',
+        beskrivelse: 'Dagpenger, AAP, egen bedrift',
+        underkategorier: [
+            {
+                tema: 'AAP',
+                tittel: 'Arbeidsavklaringspenger (AAP)',
+                skjemaveileder: false,
+                digital: false,
+                bareAnke: false
+            },
+            {
+                tema: 'DAG',
+                tittel: 'Dagpenger',
+                skjemaveileder: false,
+                digital: false,
+                bareAnke: false
+            }
+        ]
+    },
+    {
+        tittel: 'Familie og barn',
+        beskrivelse: 'Barnetrygd, foreldrepenger, pleie',
+        underkategorier: [
+            {
+                tema: 'FOR',
+                tittel: 'Foreldrepenger, engangsstønad og svangerskapspenger',
+                skjemaveileder: false,
+                digital: true,
+                bareAnke: false
+            }
+        ]
+    }
+];
+
+export const isValidKategori = (kategori: string): boolean => {
+    const foundKategori = KLAGE_ELLER_ANKE_TEMAER.find(kat => kat.tittel === kategori);
+
+    return foundKategori !== undefined;
+};
+
+export const temaHasDigitalForm = (kategori: string, tema: string) => {
+    const foundKategori = KLAGE_ELLER_ANKE_TEMAER.find(kat => kat.tittel === kategori);
+
+    if (foundKategori === undefined) {
+        throw Error;
+    } else {
+        return foundKategori.underkategorier.find(t => t.tema === tema && t.digital) !== undefined;
```

Comment:
I stedet for å sjekke om `find()` returnerer `undefined` kan man bruke `some()`, som allerede returnerer `boolean`.